CONTRIBUTING.md

```diff
@@ -216,6 +215,58 @@ items. In addition, use the following review explanations:
 
 If you open a PR in ICS, it is mandatory to update the relevant documentation in `/docs`.
 
+### Changelog
+
+To manage and generate our changelog, we currently use [unclog](https://github.com/informalsystems/unclog).
+
+Every PR with types `fix`, `feat`, `deps`, and `refactor` should include a file 
+`.changelog/unreleased/${section}/[${component}/]${pr-number}-${short-description}.md`,
+where:
+
+- `section` is one of 
+  `dependencies`, `improvements`, `features`, `bug-fixes`, `state-breaking`, `api-breaking`, 
+  and _**if multiple apply, create multiple files**_;
```

Comment:
With the same name and contents?

Reply:
Not necessarily. I'll clarify it in the doc.